state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -505,6 +508,14 @@ pub mod pallet {
 		ConstU32<DEFAULT_MAX_SWAP_REQUEST_DURATION_BLOCKS>,
 	>;
 
+	/// The minimum chunk size for DCA swaps. The number of chunks of a DCA swap will be reduced
+	/// so that the chunk size is greater than or equal to this value. Setting to zero will disable
+	/// the check for that asset.
+	#[pallet::storage]
+	#[pallet::getter(fn minimum_chunk_size)]
+	pub type MinimumChunkSize<T: Config> =
+		StorageMap<_, Twox64Concat, Asset, AssetAmount, ValueQuery>;
```

Comment:
Doesn't really need to be stored as an `AssetAmount`. `u32` is plenty, and then we don't have the confusion of returning a NumberOrHex from the rpc.

Reply:
Some assets like ETH have 18 decimal places. Even if we set this so something small like 0.001 ETH, it would still be a large enough number that it won't fit in u32, right?

Reply:
Yes, ignore me. For some reason I was confusing chunk size with number of chunks. AssetAmount makes total sense.